packages/create-next-app/create-app.ts

```diff
@@ -203,6 +203,7 @@ export async function createApp({
      */
     const packageJson = {
       name: appName,
+      version: '0.1.0',
```

Comment:
How about `0.0.0-dev` or something else that does not look like a releaseable version? If a bug only exists in old Yarn versions, what about adding this line only in some cases?